<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="8">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0,0,0">
      <property name="spacing">
       <number>4</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0,0">
        <property name="spacing">
         <number>5</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="spacing">
           <number>5</number>
          </property>
          <item>
           <widget class="QGroupBox" name="data1_bias">
            <property name="font">
             <font>
              <pointsize>7</pointsize>
             </font>
            </property>
            <property name="acceptDrops">
             <bool>true</bool>
            </property>
            <property name="title">
             <string>data1_bias</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <layout class="QGridLayout" name="gridLayout_3">
             <item row="0" column="0">
              <widget class="QLabel" name="data1">
               <property name="minimumSize">
                <size>
                 <width>80</width>
                 <height>50</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>100</width>
                 <height>100</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <pointsize>15</pointsize>
                </font>
               </property>
               <property name="acceptDrops">
                <bool>false</bool>
               </property>
               <property name="autoFillBackground">
                <bool>false</bool>
               </property>
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <property name="text">
                <string>float</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
               <property name="margin">
                <number>-6</number>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="data2_bias">
            <property name="font">
             <font>
              <pointsize>7</pointsize>
             </font>
            </property>
            <property name="acceptDrops">
             <bool>true</bool>
            </property>
            <property name="title">
             <string>data2_bias</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <layout class="QGridLayout" name="gridLayout_5">
             <item row="0" column="0">
              <widget class="QLabel" name="data2">
               <property name="minimumSize">
                <size>
                 <width>80</width>
                 <height>50</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>100</width>
                 <height>100</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <pointsize>15</pointsize>
                </font>
               </property>
               <property name="acceptDrops">
                <bool>false</bool>
               </property>
               <property name="autoFillBackground">
                <bool>false</bool>
               </property>
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <property name="text">
                <string>float</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
               <property name="margin">
                <number>-6</number>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="data3_bias">
            <property name="font">
             <font>
              <pointsize>7</pointsize>
             </font>
            </property>
            <property name="acceptDrops">
             <bool>true</bool>
            </property>
            <property name="title">
             <string>data3_bias</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <layout class="QGridLayout" name="gridLayout_7">
             <item row="0" column="0">
              <widget class="QLabel" name="data3">
               <property name="minimumSize">
                <size>
                 <width>80</width>
                 <height>50</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>100</width>
                 <height>100</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <pointsize>15</pointsize>
                </font>
               </property>
               <property name="acceptDrops">
                <bool>false</bool>
               </property>
               <property name="autoFillBackground">
                <bool>false</bool>
               </property>
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <property name="text">
                <string>float</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
               <property name="margin">
                <number>-6</number>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_9">
          <property name="spacing">
           <number>5</number>
          </property>
          <item>
           <widget class="QGroupBox" name="data4_bias">
            <property name="font">
             <font>
              <pointsize>7</pointsize>
             </font>
            </property>
            <property name="acceptDrops">
             <bool>true</bool>
            </property>
            <property name="title">
             <string>data4_bias</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <layout class="QGridLayout" name="gridLayout_11">
             <item row="0" column="0">
              <widget class="QLabel" name="data4">
               <property name="minimumSize">
                <size>
                 <width>80</width>
                 <height>50</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>100</width>
                 <height>100</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <pointsize>15</pointsize>
                </font>
               </property>
               <property name="acceptDrops">
                <bool>false</bool>
               </property>
               <property name="autoFillBackground">
                <bool>false</bool>
               </property>
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <property name="text">
                <string>float</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
               <property name="margin">
                <number>-6</number>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="data5_bias">
            <property name="font">
             <font>
              <pointsize>7</pointsize>
             </font>
            </property>
            <property name="acceptDrops">
             <bool>true</bool>
            </property>
            <property name="title">
             <string>data5_bias</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <layout class="QGridLayout" name="gridLayout_12">
             <item row="0" column="0">
              <widget class="QLabel" name="data5">
               <property name="minimumSize">
                <size>
                 <width>80</width>
                 <height>50</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>100</width>
                 <height>100</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <pointsize>15</pointsize>
                </font>
               </property>
               <property name="acceptDrops">
                <bool>false</bool>
               </property>
               <property name="autoFillBackground">
                <bool>false</bool>
               </property>
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <property name="text">
                <string>float</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
               <property name="margin">
                <number>-6</number>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="data6_bias">
            <property name="font">
             <font>
              <pointsize>7</pointsize>
             </font>
            </property>
            <property name="acceptDrops">
             <bool>true</bool>
            </property>
            <property name="title">
             <string>data6_bias</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <layout class="QGridLayout" name="gridLayout_13">
             <item row="0" column="0">
              <widget class="QLabel" name="data6">
               <property name="minimumSize">
                <size>
                 <width>80</width>
                 <height>50</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>100</width>
                 <height>100</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <pointsize>15</pointsize>
                </font>
               </property>
               <property name="acceptDrops">
                <bool>false</bool>
               </property>
               <property name="autoFillBackground">
                <bool>false</bool>
               </property>
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <property name="text">
                <string>float</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
               <property name="margin">
                <number>-6</number>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="spacing">
           <number>5</number>
          </property>
          <item>
           <widget class="QGroupBox" name="data7_bias">
            <property name="font">
             <font>
              <pointsize>7</pointsize>
             </font>
            </property>
            <property name="acceptDrops">
             <bool>true</bool>
            </property>
            <property name="title">
             <string>data7_bias</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <layout class="QGridLayout" name="gridLayout_15">
             <item row="0" column="0">
              <widget class="QLabel" name="data7">
               <property name="minimumSize">
                <size>
                 <width>80</width>
                 <height>50</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>100</width>
                 <height>100</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <pointsize>15</pointsize>
                </font>
               </property>
               <property name="acceptDrops">
                <bool>false</bool>
               </property>
               <property name="autoFillBackground">
                <bool>false</bool>
               </property>
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <property name="text">
                <string>float</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
               <property name="margin">
                <number>-6</number>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="data8_bias">
            <property name="font">
             <font>
              <pointsize>7</pointsize>
             </font>
            </property>
            <property name="acceptDrops">
             <bool>true</bool>
            </property>
            <property name="title">
             <string>data8_bias</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <layout class="QGridLayout" name="gridLayout_14">
             <item row="0" column="0">
              <widget class="QLabel" name="data8">
               <property name="minimumSize">
                <size>
                 <width>80</width>
                 <height>50</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>100</width>
                 <height>100</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <pointsize>15</pointsize>
                </font>
               </property>
               <property name="acceptDrops">
                <bool>false</bool>
               </property>
               <property name="autoFillBackground">
                <bool>false</bool>
               </property>
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <property name="text">
                <string>float</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
               <property name="margin">
                <number>-6</number>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="data9_bias">
            <property name="font">
             <font>
              <pointsize>7</pointsize>
             </font>
            </property>
            <property name="acceptDrops">
             <bool>true</bool>
            </property>
            <property name="title">
             <string>data9_bias</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <layout class="QGridLayout" name="gridLayout_16">
             <item row="0" column="0">
              <widget class="QLabel" name="data9">
               <property name="minimumSize">
                <size>
                 <width>80</width>
                 <height>50</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>100</width>
                 <height>100</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <pointsize>15</pointsize>
                </font>
               </property>
               <property name="acceptDrops">
                <bool>false</bool>
               </property>
               <property name="autoFillBackground">
                <bool>false</bool>
               </property>
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <property name="text">
                <string>float</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
               <property name="margin">
                <number>-6</number>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="vert_divider">
        <property name="minimumSize">
         <size>
          <width>3</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>3</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 0, 0);</string>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3" stretch="1,0,2,2,2,0">
        <property name="spacing">
         <number>5</number>
        </property>
        <property name="sizeConstraint">
         <enum>QLayout::SetNoConstraint</enum>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QGroupBox" name="serialport_title1">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>80</height>
           </size>
          </property>
          <property name="title">
           <string>Data Serial Port</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <widget class="QLabel" name="serial_handler_output">
             <property name="font">
              <font>
               <pointsize>20</pointsize>
              </font>
             </property>
             <property name="text">
              <string>-</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="button_test">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>3000</width>
            <height>40</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Cascadia Code</family>
            <pointsize>15</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>TEST PLATFORM</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="grade">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>80</height>
           </size>
          </property>
          <property name="title">
           <string>Grade</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="0" column="0">
            <widget class="QLabel" name="grade_data">
             <property name="font">
              <font>
               <pointsize>20</pointsize>
              </font>
             </property>
             <property name="text">
              <string>-</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="parallelism">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>80</height>
           </size>
          </property>
          <property name="title">
           <string>Parallelism (&lt;= 35 micron for pass)</string>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="parallelism_data">
             <property name="font">
              <font>
               <pointsize>20</pointsize>
              </font>
             </property>
             <property name="text">
              <string>-</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="identifier">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>80</height>
           </size>
          </property>
          <property name="title">
           <string>Build Platform Identifier Code</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_6">
           <item row="0" column="0">
            <widget class="QLabel" name="identifier_data">
             <property name="font">
              <font>
               <pointsize>20</pointsize>
              </font>
             </property>
             <property name="text">
              <string>-</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QPushButton" name="button_clear">
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>50</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>1000</width>
              <height>40</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Cascadia Code</family>
              <pointsize>15</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>CLEAR</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="button_connect_scanner">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>50</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>1000</width>
              <height>40</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Cascadia Code</family>
              <pointsize>15</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>SCAN</string>
            </property>
            <property name="autoDefault">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
